flink-libraries/flink-state-processing-api/src/test/java/com/example/state/writer/job/schema/PojoData.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.state.writer.job.schema;
+
+/** Example POJO data to deserialize state. */
+public class PojoData {
+    private Long privateLong;
+    public Long publicLong;
+
+    public Long getPrivateLong() {
+        return privateLong;
+    }
+
+    public void setPrivateLong(Long privateLong) {
+        this.privateLong = privateLong;
```

Comment:
I am not sure what this example is showing .  `publicLong` is defined but not referenced at all.

Reply:
The state contains this POJO serialized. It shows that it can read private members too.